CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.6.1
+
+- Feature: [Improved serialization failure messages](https://github.com/absinthe-graphql/absinthe/pull/1033)
```

Comment:
@binaryseed I'm somewhat inclined to push for PRs to add their own changelog entries when we think one is applicable. This ensures they actually happen and involves the PR author in the description of the change.

Reply:
The only tricky bit is the lack of a PR # before submitting :)

Reply:
That is true, I had to add mine as a commit after opening the PR. Still, that doesn't seem like too painful of a requirement.